lib/elixometer.ex

```diff
@@ -174,13 +174,13 @@ defmodule Elixometer do
 
   def get_metric_value(metric_name) do
     metric_name
-    |> to_atom_list
+    |> String.split(".")
```

Comment:
https://github.com/pinterest/elixometer/pull/74/commits/7061c286c29843024ee320ce4fc70c6b92d0c170 - is that what you meant?